utils/docker-ce/files/dockerd.init

```diff
@@ -59,6 +59,16 @@ start_service() {
 	procd_close_instance
 }
 
+reload_service() {
+	process_config
+	#SIGHUP is used by dockerd to do init.d reload
```

Comment:
I'm not sure what this comment means.
The link below suggests that `SIGHUP` will cause the daemon to reload the configuration.
https://docs.docker.com/config/daemon/#enable-debugging

Reply:
I copied this from the keepaliveed package.
But you're right.
I'll delete it.